agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaEventServiceImpl.java

```diff
@@ -228,6 +233,7 @@ public Event createEvent(Event event,
     reminderService.saveEventReminders(createdEvent, reminders, userIdentityId);
     attendeeService.saveEventAttendees(createdEvent, attendees, userIdentityId, sendInvitation, false);
 
+    Utils.broadcastEvent(listenerService, "exo.agenda.event.added", eventId, 0);
```

Comment:
You didn't used `sendInvitation` flag. In fact, this listener shouldn't be triggered for notifications. May be for analytics only.

Reply:
= i will change it with Utils.broadcastEvent(listenerService, "exo.agenda.event.added", eventId, sendInvitation);